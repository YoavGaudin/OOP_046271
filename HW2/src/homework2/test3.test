# test one child, two childs

CreateGraph graph1
CreateNode n1 5
CreateNode n2 5
CreateNode n3 5

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

AddEdge graph1 n1 n2
ListChildren graph1 n1
# test adding the same edge twice has no effect
AddEdge graph1 n1 n2
ListChildren graph1 n1

AddEdge graph1 n1 n3
ListChildren graph1 n1