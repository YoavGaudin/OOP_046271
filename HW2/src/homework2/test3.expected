# test one child, two childs

created graph graph1
created node n1 with cost 5
created node n2 with cost 5
created node n3 with cost 5

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

added edge from n1 to n2 in graph1
the children of n1 in graph1 are: n2
# test adding the same edge twice has no effect
added edge from n1 to n2 in graph1
the children of n1 in graph1 are: n2

added edge from n1 to n3 in graph1
the children of n1 in graph1 are: n2 n3
